content/en/platform/corda/5.2/deploying-operating/deployment/upgrading/_index.md

```diff
@@ -56,83 +58,353 @@ For information about rolling back the Corda 5.0 to Corda {{< version-num >}} up
 ## Scale Down the Running Corda Worker Instances
 
 {{< important >}}
-Scaling down the Corda 5.0 workers results in Corda becoming unresponsive until you launch the Corda {{< version-num >}} workers. You must first test the database migration as described in [Test the Migration](#test-the-migration) and also ensure that you are familiar with the rest of the upgrade process to incur minimum Corda downtime.
+Scaling down the Corda 5.1 workers results in Corda becoming unresponsive until you launch the Corda {{< version-num >}} workers. You must first test the database migration as described in [Test the Migration](#test-the-migration) and also ensure that you are familiar with the rest of the upgrade process to incur minimum Corda downtime.
 {{< /important >}}
 
 You can scale down the workers using any tool of your choice. For example, the run the following commands if using `kubectl`:
 
 ```shell
 kubectl scale --replicas=0 deployment/corda-crypto-worker -n <corda_namespace>
 kubectl scale --replicas=0 deployment/corda-db-worker -n <corda_namespace>
+kubectl scale --replicas=0 deployment/corda-flow-mapper-worker -n <corda_namespace>
 kubectl scale --replicas=0 deployment/corda-flow-worker -n <corda_namespace>
 kubectl scale --replicas=0 deployment/corda-membership-worker -n <corda_namespace>
 kubectl scale --replicas=0 deployment/corda-p2p-gateway-worker -n <corda_namespace>
 kubectl scale --replicas=0 deployment/corda-p2p-link-manager-worker -n <corda_namespace>
+kubectl scale --replicas=0 deployment/corda-persistence-worker -n <corda_namespace>
 kubectl scale --replicas=0 deployment/corda-rest-worker -n <corda_namespace>
+kubectl scale --replicas=0 deployment/corda-token-selection-worker -n <corda_namespace>
+kubectl scale --replicas=0 deployment/corda-uniqueness-worker -n <corda_namespace>
+kubectl scale --replicas=0 deployment/corda-verification-worker -n <corda_namespace>
 ```
 
 If you are scripting these commands, you can wait for the workers to be scaled down using something similar to the following:
 ```shell
-while [ "$(kubectl get pods --field-selector=status.phase=Running -n corda | grep worker | wc -l | tr -d ' ')" != 0 ]
+while [ "$(kubectl get pods --field-selector=status.phase=Running -n <corda_namespace> | grep worker | wc -l | tr -d ' ')" != 0 ]
 do
   sleep 1
 done
 ```
 
 ## Migrate the Corda Cluster Database
 
-To migrate the database schemas, do the following:
+To migrate the cluster database schemas, do the following:
 
 1. Generate the required SQL scripts using the `spec` sub-command of the <a href = "../../reference/corda-cli/database.md"> Corda CLI `database` command</a>. For example:
 
    {{< tabs name="database">}}
    {{% tab name="Bash" %}}
    ```sh
```

Comment:
Minor but using `/sql_updates` here is a little unusual as it means you are creating them in the root directory. Probably `./sql_updates` is a more realistic example?

I see below you suggest to apply them from `./sql_updates` so in order for these two parts to match you would have to use `./sql_updates` everywhere too.